supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/MachineConstraints.scala

```diff
@@ -8,10 +8,18 @@ import mouse.all._
 import org.slf4j.Logger
 import wdl4s.parser.MemoryUnit
 import wom.format.MemorySize
+import scala.math.BigDecimal.RoundingMode.CEILING
+
+case class TypeCastMismatch(message: String) extends Exception(message)
 
 object MachineConstraints {
   implicit class EnhancedInformation(val information: MemorySize) extends AnyVal {
-    def asMultipleOf(factor: MemorySize): MemorySize = MemorySize(factor.amount * (information.bytes / factor.bytes).ceil, factor.unit)
+    def ceil(a: BigDecimal): Double = {
```

Comment:
It's quite strange that a method called `ceil` returns the value converted from BigDecimal to Double.
Consider a more appropriate name.